templates/about/phones.md

```diff
@@ -0,0 +1,135 @@
+title: Phones
+---
+# Phones
+
+The Phone Team (aka POC) provides a comprehensive phone service at EMF.
+
+## Getting Started
+
+You can pick a 4-digit extension number to use, and call anyone on site via it. Create your number in the  [Electromagnetic Phonebook](https://phones.emfcamp.org)
+
+Firstly you will need to register an account with a username, email & password. We send an email validation message from [poc@emfcamp.org](mailto:poc@emfcamp.org) so make sure you’ve clicked that before you can login.
+
+### Registering a Number
+
+Once logged in cick Add a Number in the top menu bar.
+
+This will then show you the Create Number form
+
+
+* **Event** will be EMF2024 (only option)
+* **Type of Service **allows you to select what sort of device you want to use this number with, see the section below for more details.
+* **Number** should be your desired 4 digit number in the range 2000-8999 or 9200-9899, some numbers may already be taken even if they do not show up in the phonebook.
+* **Description** is a free text label for how you want your entry to appear in the phonebook.
+* **Public Phonebook** allows you to choose if you want to be in the public listing or not, if not only the Phone Team can see your entry.
+* The final field will vary depending on Type of Service, usually you can leave this as it is auto-populated and not editable, you don’t need to make a note of this information you can view it later.
+
+Click Submit and your number will be created.
+
+NOTE:
+
+If you had a Permanent Number Reserved in the Eventphone system and used this on DECT at EMF2022 then we have kept your reservation, please email [poc@emfcamp.org](mailto:poc@emfcamp.org) once you have created an account in our system for us to transfer it over to you.
+
+
+### Managing your Numbers
+
+The My Numbers link will show you the numbers under your account,
+
+Next to each number is 3 buttons.
+
+**Details** will show you how to use that number with specific instructions for the Type of Service you are using.
+
+**Modify** lets you amend the Description in the phonebook, change the visibility in the phonebook and set a Fallback Number.
+
+The** Falback Number **is a number that will be called if the main number doesn’t answer the call within 30sec or is unavailable. 
+
+
+## Types of Service
+
+
+### DECT 
+
+There's site-wide DECT phone coverage
+
+You can bring your own DECT telephone and join the network, enabling you to call other EMF participants and interactive services for free.
+
+If you're buying a DECT phone and don't know what to get, buy pretty much any Gigaset DECT handset, for example:
+
+	Gigaset A170, £18 on Amazon [https://amzn.to/3yaJ5EG](https://amzn.to/3yaJ5EG)
```

Comment:
This is... a Matt Gray affiliate link?

Reply:
LOL I just copied that from the Wiki that Matt put on, do we have an EMF affiliate? if so we should possibly use that but equally I'm fine with Matt getting the 50p as he did the research

Reply:
Can we just make this a non-affiliate link for the moment

Reply:
Oh shit I had no idea that the Amazon short urls had my affiliate in them, sorry!
If it’s any consolation I’ve never made any money off it, and don’t even know how to look at the (depersonalised) data!